prime-angular-frontend/src/app/modules/enrolment/pages/overview/overview.component.ts

```diff
@@ -218,7 +222,18 @@ export class OverviewComponent extends BaseEnrolmentPage implements OnInit {
       certificate: !enrolment.certifications?.length,
       certificateOrOboSite: !enrolment.certifications?.length && !enrolment.oboSites?.length,
       deviceProvider: isDeviceProvider && !hasDeviceProviderIdentifier,
-      deviceProviderOrOboSite: (isDeviceProvider && !hasDeviceProviderIdentifier) && !enrolment.oboSites?.length
+      deviceProviderOrOboSite: (isDeviceProvider && !hasDeviceProviderIdentifier) && !enrolment.oboSites?.length,
+      missingPharmaNetId: this.isMissingPharmaNetId(enrolment.certifications)
     };
   }
+
+  private isMissingPharmaNetId(certifications: CollegeCertification[]): boolean {
+    return certifications.every((cert: CollegeCertification) => {
+      const prescriberIdType = this.configService.getPrescriberIdType(cert.licenseCode);
+      if (prescriberIdType === PrescriberIdTypeEnum.Mandatory) {
+        return cert.practitionerId === null;
+      }
+      else { return false; }
+    });
+  }
```

Comment:
Is it possible to run the existing validation that we have on the certifications page rather than checking for this _specific_ failure case?

Reply:
Those validators work on a `AbstractControl` so AFAIK, they can't be repurposed for this ...

```
      this.formUtilsService.setValidators(this.practitionerId, [
        Validators.required,
        FormControlValidators.numeric,
        FormControlValidators.requiredLength(5)
      ]);      
```

Reply:
I think I miss guided you a little, the only downside to going this route is currently we just display this toast

![image](https://user-images.githubusercontent.com/59669288/153525655-5d4d9d45-9ab6-42fe-9a63-14f725309ef4.png)

that doesn't really help to give the user and idea of where the error is, but we could intercept that. Im going to play around with something tonight.